front/lib/api/assistant/conversation.ts

```diff
@@ -309,6 +309,8 @@ async function createOrUpdateParticipation({
       },
     });
     if (participant) {
+      participant.changed("updatedAt", true);
```

Comment:
We want udpatedAt to be updated even if the record has not changed because the action is the same

Reply:
Neat but it does 2 async calls we can likely do just one right?